module/run_component_test.go

```diff
@@ -0,0 +1,224 @@
+package module
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/onflow/flow-go/module/irrecoverable"
+	"github.com/stretchr/testify/require"
```

Comment:
`module/run_component_test.go:10: File is not `goimports`-ed with -local github.com/onflow/flow-go (goimports)`